src/types.ts

```diff
@@ -59,6 +59,14 @@ export enum UserRole {
   CLIENT = "CLIENT"
 }
 
+// clerk
+export enum ClerkSortOrder {
+  ASC_CREATED_AT = "+created_at",
+  ASC_UPDATED_AT = "+updated_at",
+  DESC_CREATED_AT = "-created_at",
+  DESC_UPDATED_AT = "-updated_at"
+}
+
```

Comment:
Could possibly move this back to the user service class as this enum doesn't look like it gets used outside of the user service.